CHANGELOG.md

```diff
@@ -9,6 +9,24 @@ NEW FEATURES:
 
     Unlike the existing `precondition` and `postcondition` blocks, Terraform will not halt execution should the scoped data block fail or error or if any of the assertions fail. 
     This allows practitioners to continually validate the state of their infrastructure outside the usual lifecycle management cycle. 
+    
+* `import` blocks for importing infrastructure: Root module authors can now use the `import` block to declare their intent that Terraform adopt an existing resource. 
+  
+    Import is now a plannable action, and is processed as part of a normal plan. Running `terraform plan` will show a summary of the resources that Terraform has planned to import, along with any other plan changes. These may include updates and replacements to the imported resource, if the configuration for that resource differs from the state of the remote system. 
+
+    The existing `terraform import` CLI command has not been modified. Running the CLI command directly imports a resource's state into the state file, while using the `import` block and running `terraform plan` will not modify the state until `terraform apply` is run.
+
+    This is an early version of the `import` block feature, for which we are actively seeking user feedback to shape future development. The `import` block currently does not support interpolation in the `id` field, which must be a string.
+```hcl
+import {
+  id = "i-12345678"           # import ID
+  to = aws_instance.example   # target resource address
+}
+```
+
+* Generating configuration for imported resources. In conjunction with the `import` block, this feature enables easy templating of configuration when importing existing resources into Terraform. A new flag `-generate-config-out=PATH` is added to `terraform plan`. When this flag is set, Terraform will generate HCL configuration for any resource included in an `import` block that does not already have associated configuration, and write it to a new file at `PATH`. This configuration will not include sensitive values. Before applying, you should review the generated configuration and edit it as necessary. This file should be committed to version control just like any other configuration file.
```

Comment:
If you didn't have a PATH and only had `-generate-config-out`, would it write the generated config to your current directory? If so, we should state that

Reply:
@kmoe If `-generate-config-out` requires a path, then i think we should modify it so that if you do not include a path, it defaults to the current directory

Reply:
The user needs to supply a file name - Terraform doesn't choose one because it needs to be one that doesn't already exist. Let's discuss elsewhere